packages/react/yarn.lock

```diff
@@ -0,0 +1,8 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
How come this change introduced a new lock file?

Reply:
@olivier-wb, it's just because I yarn installed, I think this should be fine to keep.